### PR TITLE
fix: Unclear test output use-array.map-to-dynamically-render-elements

### PR DESCRIPTION
**Pull request by kundan59**
The second child of MyToDoList should be a br(new line) element in use-array.map-to-dynamically-render-elements of React (Frontend Libraries)

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md). 
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

Closes #XXXXX

### CODE REVIEW

File: curriculum/challenges/english/03-front-end-libraries/react/use-array.map-to-dynamically-render-elements.english.md
Comment (ojeytonwilliams) [inline curriculum/challenges/english/03-front-end-libraries/react/use-array.map-to-dynamically-render-elements.english.md]:
```suggestion
  - text: The third child of <code>MyToDoList</code> should be a <code>button</code> element.
```
As I mentioned in https://github.com/freeCodeCamp/freeCodeCamp/issues/38170#issuecomment-582799717 the button is the third child, and the second child is a `<br/>` element.  If anything needs changing, it's the seed.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

**Review by RandellDawson — APPROVED** [review]
LGTM @ojeytonwilliams Merge when ready.

**Review by ojeytonwilliams — APPROVED** [review]
Perfect, thanks @kundan59!

File: curriculum/challenges/english/03-front-end-libraries/react/use-array.map-to-dynamically-render-elements.english.md
Comment (ojeytonwilliams) [inline curriculum/challenges/english/03-front-end-libraries/react/use-array.map-to-dynamically-render-elements.english.md]:
```suggestion
  - text: The second child of <code>MyToDoList</code> should be a <code>br</code> element.
```
Sorry, missed this when reviewed before.

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/03-front-end-libraries/react/use-array.map-to-dynamically-render-elements.english.md
Comment (kundan59) [reply curriculum/challenges/english/03-front-end-libraries/react/use-array.map-to-dynamically-render-elements.english.md]: @ojeytonwilliams sry I missed that

Comment (ojeytonwilliams) [reply curriculum/challenges/english/03-front-end-libraries/react/use-array.map-to-dynamically-render-elements.english.md]: No worries, that's what reviews are for.

**Review by ojeytonwilliams — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
I'm a bit confused about why this isn't triggering travis.  I'll try force pushing and see where that gets us.

**Comment by ojeytonwilliams** [comment]
Nevermind, I just realised I'd be pushing to your master branch. Even if I *could* do that, I don't think I should.  

So, @kundan59 could you force push and see if that triggers Travis?  Also, in future, please create a new branch instead of using master.  Using master makes it harder for reviewers to help fix problems and means it's more difficult to make sure your own fork doesn't get into a weird state.

**Comment by kundan59** [comment]
@ojeytonwilliams  I tried push force, as everything up to date it didn't affect anything. I will take care of that in future to have work on new branch.

**Comment by ojeytonwilliams** [comment]
I'm not sure why Travis didn't pick up on this, but I can't merge it until it does.  What I suggest is the following.  Create a new branch, i.e. `git checkout -b fix/dynamic-challenge`, and create a new pull request using that.  If it works we can merge the new one.

Sorry for the hassle!

**Comment by kundan59** [comment]
@ojeytonwilliams I have raised new PR for the same [](https://github.com/freeCodeCamp/freeCodeCamp/pull/38172) could you please review it and close this PR.

**Comment by raisedadead** [comment]
Hi @kundan59 did you get a chance to reach out to TravisCI support? Do you need any help?

**Comment by kundan59** [comment]
Hi, @raisedadead I am able to reach Travis CI Support and they suggested me to activate the debug feature for this kundan59/freeCodeCamp repository. Will inform you soon when I complete this feature.

**Comment by kundan59** [comment]
Hi, @raisedadead even I tried to debug build in VM locally but I am not able to resolve this issue. The build is failed because of Lint Javascript fails.
Screenshot from 2020-02-10 21-11-42

**Comment by raisedadead** [comment]
I think we are talking about two different issues here:

1. Travis CI itself is not triggering for all the PRs from your username.

   image


2. There is an actual linting issue (which may become evident if issue no.1 above is resolved first)

While you try to figure out the first issue, let's see if there is anything weird going on with the linter.

On that note, it may be useful to get your fork rebased to the latest versions of the master. Please reach out to [contributors chat room](https://gitter.im/FreeCodeCamp/Contributors) for help with that.

**Comment by raisedadead** [comment]
I just happened to notice that this PR is also coming from the master branch of your fork which makes things even more complicated. Please reach out to the help room.

**Pull request closed without merging**
